### DISCUSSION TITLE
关于用户查询命令和管理员查询命令

### DISCUSSION THREAD

**Discussion by laz7cat** [liusy58/NJU_AdvancedProgramming_SP22#73] [Q&A] [answered]
ppt里说管理员和买家查看商品列表的命令都是select * from commodity，但是管理员可以看到下架状态的商品，用户却看不到。这是不是有点不合理呢？可否修改查看的命令，比如说改成select * from commodity where isSelling = true之类的？

**Comment by binbin00** [comment, marked as answer]
https://github.com/liusy58/NJU_AdvancedProgramming_SP22/discussions/27
